### PR TITLE
Configuration for Proxify

### PR DESCRIPTION
**Pull request by reinhardlinardi**
I add a configuration file for `proxify` (named *proxify.conf*). You can put in your home directory and `proxify` will never prompt you again for host, port, and username. It will prompt you only for password. I updated the README and add comments inside the script too.

I think everybody tired of typing host, port, and username for proxify :D

### CODE REVIEW

File: proxify/proxify [hunk 1]
```diff
@@ -1,81 +1,128 @@
...
 	fi
 
-	echo "$base"
+	echo "$base"  # return result
 }
 
 # Main
 
-base="$(gen_base)"
+conf_file=$(echo ~/proxify.conf)  # path of configuration file
```
Comment (nieltg) [inline proxify/proxify]: Please rename this to `.proxify.conf`, so users don't get stray config. files around.

**Comment by nieltg** [comment]
Hi Reinhard,

Well, I think there is one thing to do.
Not everyone likes to have a stray config. file in on their home directory.
So, I think it's better to hide it from the user.

I've reviewed your code. Consider looking at it.
Thank you very much.

**Review by nieltg — CHANGES_REQUESTED** [review]
Hi Reinhard,

Well, I think there is one thing to do.
Not everyone likes to have a stray config. file in on their home directory.
So, I think it's better to hide it from the user.

I've reviewed your code. Consider looking at it.
Thank you very much.

**Comment by reinhardlinardi** [comment]
Done.

**Review by nieltg — APPROVED** [review]

**Comment by nieltg** [comment]
Thank you very much, Reinhard!

**Comment by reinhardlinardi** [comment]
:)

**Pull request merged**